reana_client/cli/files.py

```diff
@@ -66,6 +67,14 @@ def files_group(ctx):
     default=None,
     help="Get URLs of output files.",
 )
+@click.option(
+    "--filter",
+    "filters",
+    multiple=True,
+    help="Filter workflow that contains certain filtering criteria. "
```

Comment:
Filter results to show only files that match certain filtering criteria such as file name or file modification date.